tools/topology/sof/tokens.m4

```diff
@@ -89,3 +89,12 @@ SectionVendorTokens."sof_tone_tokens" {
 SectionVendorTokens."sof_effect_tokens" {
 	SOF_TKN_EFFECT_TYPE			"900"
 }
+
+SectionVendorTokens."sof_detect_tokens" {
+	SOF_TKN_DETECT_ITEM1			"1000"
```

Comment:
can we find a better name? what does this mean?

Reply:
This is intended for some detection customisation, but can be dropped atm since we are not using it.

Reply:
removed now